src/main/java/io/vlingo/wire/fdx/bidirectional/rsocket/RSocketClientChannel.java

```diff
@@ -59,7 +59,7 @@ public void close() {
     if (this.channelSocket != null && !this.channelSocket.isDisposed()) {
       try {
         this.channelSocket.dispose();
-      } catch (final Throwable t) {
+      } catch (final Exception t) {
```

Comment:
Do we really want to propagate an unchecked exception if the channel was not closed?

Reply:
Unchecked exceptions are caught. The previous version would catch `Error` as well as `Exception`. Do we need to catch `Error`?

Reply:
You are right. I should have said `Errors`.